cli.ts

```diff
@@ -181,6 +182,29 @@ function _task(task: string): void {
   }
 }
 
+function _ensureJsFiles(paths: string[]) {
+  let errors = 0;
+  for (const path of paths) {
+    if (
+      path.includes(".") && !(path.endsWith(".js") || path.endsWith(".ts"))
```

Comment:
Could you 
- include `.jsx` and `.tsx` as well?
- use `extname` from `std/path` to check file extensions?

Reply:
done